lib/plugins/globalid.js

```diff
@@ -4,8 +4,8 @@
 
 // Modules
 var jsonp = require('jsonp')
-var _ = require('../utils/lodash')
-var invariant = require('invariant')
+var assert = require('../utils/misc').assert
+var noop = require('../utils/misc').noop
```

Comment:
Would be interested into size comparision between declaring noop manually when used compared to the webpacked size of requiring it from lodash + requirer boilerplate code